src/test/IsLessThanOrEqualToSumOf.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from 'chai'
+import { IsLessThanOrEqualToSumOf, IsLessThanOrEqualToSumOfConstraint } from '../main/isLessThanOrEqualToSumOf'
+import { validateSync, ValidationArguments } from 'class-validator'
+
+const DEFAULT_SUM = 30
+const DEFAULT_VALUE = 200
+
+class LessThanOrEqualToSumTest {
+  @IsLessThanOrEqualToSumOf('defaultSum', 'targetValue')
+  value: any
+  targetValue: number
+  defaultSum: number
+
+  constructor (value: any, sum: any, lessThan: any) {
+    this.value = value
+    this.defaultSum = sum
+    this.targetValue = lessThan
+  }
+}
+
+describe('IsLessThanOrEqualToSumOf', () => {
+
+  describe('validate', () => {
+
+    describe('should return true when ', () => {
```

Comment:
You do not need to add whitespace at the end. Reporter will take care of formatting test case names in a readable way. Please remove whitespace here and in other places.